pkg/domain/infra/abi/containers.go

```diff
@@ -406,6 +406,12 @@ func (ic *ContainerEngine) ContainerInspect(ctx context.Context, namesOrIds []st
 
 		inspect, err := ctr.Inspect(options.Size)
 		if err != nil {
+			// ErrNoSuchCtr is non-fatal, other errors will be
+			// treated as fatal.
+			if errors.Cause(err) == define.ErrNoSuchCtr {
```

Comment:
Please use errors.Is()

Reply:
There are over 100 such comparisons in Podman (see `vgrep "== define.Err"`) and dozens in this file. If we open this box in this PR, I fear it'll never merge :^)

Reply:
Every new code should use errors.Is() IMO. We have to move to errors.IS() at same point so please don't add more errors.Cause() calls.
https://github.com/containers/podman/pull/11341#issuecomment-908122635

Reply:
I agree in general but it should be consistent. There's the same check a couple of lines above. So where do we draw the line? If adding a new `errors.Is()` should the entire function be changed? Should the call stack be changed, the file, the package?

In this case we're altering existing code and that should be consistent with the location IMO.

Reply:
Note: I think at least the entire function should in such case be changed.

Reply:
I just don't want new errors.Cause() calls. If you worry about consistency, then yes please change the entire function.
Anyway this is not a blocker for me, if other maintainers are fine with it we can merge.